tests/SprykerSdkTest/Zed/AppSdk/Communication/Console/CreateManifestConsoleTest.php

```diff
@@ -32,12 +33,44 @@ class CreateManifestConsoleTest extends Unit
      */
     public function testCreateManifestConsole(): void
     {
+        // Arrange
         $commandTester = $this->tester->getConsoleTester(new CreateManifestConsole());
 
+        $filePath = sprintf('%s/config/app/manifest/', vfsStream::setup('root')->url());
+
         // Act
-        $commandTester->execute([CreateManifestConsole::MANIFEST_NAME => 'Manifest', '--' . CreateManifestConsole::OPTION_MANIFEST_PATH => 'config/example/en_US.json'], ['verbosity' => OutputInterface::VERBOSITY_VERBOSE]);
+        $commandTester->execute(
+            [
+                CreateManifestConsole::MANIFEST_NAME => 'Manifest',
+                CreateManifestConsole::MANIFEST_LOCALE => 'en_US',
+                '--' . CreateManifestConsole::OPTION_MANIFEST_PATH => $filePath,
+            ],
+            ['verbosity' => OutputInterface::VERBOSITY_VERBOSE],
+        );
 
         // Assert
         $this->assertSame(AbstractConsole::CODE_SUCCESS, $commandTester->getStatusCode());
     }
+
+    /**
+     * @return void
+     */
+    public function testCreateManifestConsoleWithFileExists(): void
+    {
+        $commandTester = $this->tester->getConsoleTester(new CreateManifestConsole());
+
```

Comment:
```suggestion
        // Arrange
        $commandTester = $this->tester->getConsoleTester(new CreateManifestConsole());
        $this->tester->haveManifestFile();
```

ManifestHelper::haveManifestFile() will create a virtual file see https://github.com/spryker-sdk/app-sdk/blob/master/tests/_support/Helper/ManifestHelper.php#L80

The file will be `en_US.json`